EARLY-DISCLOSURE.md

```diff
@@ -62,16 +62,18 @@ email address, it must be a corporate address owned by the organization.
 3. Have a user base not limited to your own organization.
 4. Be a participant and active contributor in the Istio community.
 5. Accept the [Embargo Policy](#embargo-policy) that is outlined above.
+6. Have a member of your organization be a participant in the Istio Product Security Working Group. This is a private meeting held alternating Wednesdays at 11am Pacific or 3pm Pacific to accommodate a world-wide community.
```

Comment:
Should we set out here the expectation that they will, on a quarterly basis, be expected to lead CVE releases during that time?